Composer/packages/client/package.json

```diff
@@ -20,6 +20,7 @@
     "@bfc/extensions": "*",
     "@bfc/indexers": "*",
     "@bfc/shared": "*",
+    "@bfcomposer/bf-lu": "^1.1.5",
```

Comment:
One question to @lei9444 , is this the latest version of bf-lu and is this the version which lubuild used? we need to make sure this version is consistent

Reply:
lubuild use 1.1.2 now

Reply:
bf-lu@1.1.2+ is some bug fix or features update, base on my feedback to Fei.

Reply:
The latest version is 1.1.6 now

Reply:
So 1.1.2 is OK for lubuild, right? get it